File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        Set<Integer> lst = new HashSet();
        for (int i=0;i < nums.length;i++){
            if (lst.contains(nums[i])){
                return true;
            }
            lst.add(nums[i]);
        }
        return false;
    }
}

File: Data Structures & Algorithms/validate-parentheses/submission-1.py
class Solution:
    def isValid(self, s: str) -> bool:
        lst = []
        m = {'(': ')',
             '{': '}',
             '[': ']'}
        e = {')': '(',
             '}': '{',
             ']': '['}
        if s == "":
            return True
        for c in s:
            if c in m:
                lst.append(c)
            if c in e:
                if len(lst) != 0 and lst.pop() == e[c]:
                    continue
                else:
                    return False
            print(lst)
            
        return len(lst) == 0